jaco_moveit_config/launch/moveit_planning_execution.launch

```diff
@@ -31,12 +31,11 @@
   <!--   - this typically includes: robot_state, motion_interface, and joint_trajectory_action nodes -->
   <!--   - replace these calls with appropriate robot-specific calls or launch files -->
   <group unless="$(arg sim)">
-    <include file="$(find jaco_driver)/launch/jaco_arm.launch" />
+    <include file="$(find jaco_driver)/launch/jaco_arm.launch">
+      <arg name="use_urdf" value="true"/>
```

Comment:
This is the new /tf publishing API, much more ROS-ified in my opinion. Kinova says the other version is depricated